armada-ths/ais
# Optional fields for student profile

Added "linkedin profile", "facebook profile" and "phone number" fields to student_profile model and implemented them in student_profile api.

To test:
* migrate
* run tests
* make a manual PUT request to localhost:8000/api/student_profile?student_id={NUMBER} with payload: json'{"nickname":"NICKNAME", "facebook_profile":"FACEBOOK", "linkedin_profile":"LINKEDIN", "phone_number":"PHONENUMBER"}'
* note: everything but the nickname is optional, so you can only send the nickname
* make a GET request (goto) localhost:8000/api/student_profile?student_id={SAME_NUMBER_AS_IN_PUT} and you should see the fields you submitted in PUT request.
